x-pack/test_serverless/functional/test_suites/observability/observability_logs_explorer/data_source_selector.ts

```diff
@@ -214,25 +214,25 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
         it('should sort the integrations list by the clicked sorting option', async () => {
           // Test ascending order
-          await PageObjects.observabilityLogsExplorer.clickSortButtonBy('asc');
-
           await retry.try(async () => {
+            await PageObjects.observabilityLogsExplorer.clickSortButtonBy('desc');
```

Comment:
Wouldn't be better here just to await until the proper button is clicked?

Reply:
The await on button click works and reflects the correct UI state. The problem is we can't determine if the network call against the button click succeeds or fails. Doing this should re-issue the network call in retry if previous one fails or takes longer.

Reply:
From what I saw directly in the UI, the call made doesn't include the sort attribute, so calling it for the proper `asc` value would be enough. I don't get why we are calling `asc` and then `desc`, and viceversa.

Reply:
>  I don't get why we are calling asc and then desc, and viceversa

Without toggling the asc/desc, it doesn't re-issue the request:

https://github.com/user-attachments/assets/e8ad6861-cc68-47a7-a863-648a1fd1f71f

With the toggle it re-issues the request on each retry:

https://github.com/user-attachments/assets/10218f96-5f80-46fc-8417-ca8128a6d1d2

> the call made doesn't include the sort attribute

If you mean the sort attribute in API call, the sort asc/desc request does include the `sort` param.

<img width="1484" alt="sort-order-fleet-installed-network-request" src="https://github.com/user-attachments/assets/5a1739e4-94cc-4e45-a661-b325d4682cd2">

Reply:
> If you mean the sort attribute in API call, the sort asc/desc request does include the sort param.

I missed it somehow 😓 

> With the toggle it re-issues the request on each retry

I think that a weird behaviour, thanks for digging into it